src/main/java/pl/schoolmanager/bean/SchoolAdminAccess.java

```diff
@@ -0,0 +1,27 @@
+package pl.schoolmanager.bean;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+import pl.schoolmanager.entity.UserRole;
+import pl.schoolmanager.repository.UserRoleRepository;
+
+@Component
+public class SchoolAdminAccess {
+	@Autowired
+	UserRoleRepository userRoleRepo;
+	
+	public boolean checkAccess(Authentication auth, long schoolAdminId) {
```

Comment:
nice oportunity to make abstract class which SchoolAdminAccess, StudentAccess, TeacherAccess will inherit